bundles/org.openhab.binding.shelly/README.md

```diff
@@ -416,8 +438,8 @@ In this case the is no real measurement based on power consumption, but the Shel
 |          |input        |Switch   |yes      |ON: Input/Button is powered, see General Notes on Channels                       |
 |          |button       |Trigger  |yes      |Event trigger, see section Button Events                                         |
 |meter     |currentWatts |Number   |yes      |Current power consumption in Watts                                               |
-|          |lastPower1   |Number   |yes      |Energy consumption for a round minute, 1 minute  ago                   |
-|          |totalKWH     |Number   |yes      |Total energy consumption in Watts since the device powered up (resets on restart)|
+|          |lastPower1   |Number   |yes      |Energy consumption for a round minute, 1 minute  ago                             |
+|          |totalKWH     |Number   |yes      |Total energy consumption in kw/h since the device powered up (resets on restart) |
```

Comment:
Please check all.
```suggestion
|          |totalKWH     |Number   |yes      |Total energy consumption in kWh since the device powered up (resets on restart) |
```

Reply:
fixed